opm/simulators/linalg/PreconditionerWithUpdate.hpp

```diff
@@ -91,6 +91,7 @@ struct GeneralPreconditionerMaker {
     virtual std::unique_ptr<
         Preconditioner<typename OriginalPreconditioner::domain_type, typename OriginalPreconditioner::range_type>>
     make() = 0;
+    virtual ~GeneralPreconditionerMaker(){};
```

Comment:
Tiny nit: The final semicolon here, on the closing brace, will emit diagnostics of the form
> ```
> Redundant ';' ...
> ```
on some compilers, at least with certain warning settings.  In the overall interest of removing warnings maybe we could drop that trailing semicolon too?

Reply:
ay. missed that :/